### ISSUE TITLE
Issues gateway version 1.8.25

### ISSUE THREAD

**Issue by ggravlingen** [ggravlingen/pytradfri#230]
Seems like a new version of the gateway software is starting to roll out globally today (2019-04-10). We haven't been able to test it yet. If you run into issues (other than the ones references [here](https://github.com/ggravlingen/pytradfri#known-issues), please post them in this issue.

**Comment by ggravlingen** [comment]
Seems to be working well, closing issue.

**Comment by caphm** [comment]
I have encountered an issue while upgrading to 1.8.25 that is not pytradfri related, but which could be falsely attributed to pytradfri by users:
Some lightbulbs (E27 white spectrum) were not available via the gateway (Tradfri app showed them as unavailable). I initially thought pytradfri was at fault, but the fault was actually in the lightbulbs' connection to the gateway. Issue was solved by cutting power to the affected bulbs once. Afterwards they were available again with no issues.
Maybe you wanna add a heads-up to this issue or the readme in case others face the same issue?
